posts/ethereum-development-tips.md

```diff
@@ -0,0 +1,260 @@
+---
+authors:
+- Louis Fournier
+- Hugo de Almeida Rodrigues
+tags:
+- Blockchain
+- Ethereum
+- Dai
+- Stablecoin
+- Solidity
+- Truffle
+- Remix
+- ZeppelinOS
+- web3
+- Metamask
+- geth
+date: 
+title: "Ethereum Development Tips"
+image: "/images/2018/09/blockchain-interns-01.png"
+---
+
+![Ethereum Development Tips](/images/2018/09/blockchain-interns-01.png)
```

Comment:
we should remove this from here now, otherwise it will be displayed twice: once as the main image, once with this inline image